scan_topic: "scan_akusense"
rp_lidar_s2: false
obstalce_count: 30
signal_distance: 0.4
signal_distance_teb: 0.45
signal_release: 0.55
